js/libs/builder.js

```diff
@@ -712,6 +712,17 @@ new Vue({
         $vm.triggerSave();
       });
     },
+    toggleAccessType: function(type, isTypeActive) {
+      this.accessRules.forEach(function(rule, index, accessRules) {
+        var typeIndex = rule.type.indexOf(type);
+
+        if (isTypeActive && typeIndex === -1) {
+          accessRules[index].type.push(type);
```

Comment:
You shouldn't be modifying the `type` array for each security rule. Instead, add a new rule to `accessRules`.

Reply:
@tonytlwu Ok, I'll change it.